src/diffusers/pipelines/controlnet/pipeline_controlnet_sd_xl.py

```diff
@@ -942,7 +952,8 @@ def __call__(
             latents = latents.to(next(iter(self.vae.post_quant_conv.parameters())).dtype)
 
         if not output_type == "latent":
-            image = self.vae.decode(latents / self.vae.config.scaling_factor, return_dict=False)[0]
+            with torch.autocast(enabled=False, device_type="cuda"):
```

Comment:
No autocast in the pipelines please. We don't recommend using autocast in the first place

Reply:
Sorry, this change has made it way from my local changes, will drop it